php/commands/src/CLI_Command.php

```diff
@@ -488,6 +488,27 @@ public function cmd_dump() {
 		echo json_encode( $this->command_to_array( EE::get_root_command() ) );
 	}
 
+	/**
+	 * Uninstalls easyengine completely along with all sites
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--yes]
+	 * : Do not prompt for confirmation.
+	 *
+	 * @subcommand self-uninstall
+	 */
+	public function self_uninstall( $args, $assoc_args ) {
+
+		EE::confirm("Are you sure you want to remove EasyEngine and all its sites(along with their data) ?", $assoc_args);
```

Comment:
We should probably mention this is an irreversible action as we won't be keeping any backup at this stage.

Reply:
Done